app/views/camps/_person_fields.haml

```diff
@@ -1,14 +1,15 @@
-.nested-fields
-  = t("activerecord.attributes.person.name")
-  = f.text_field :name, class: 'form-control'
-  = t("activerecord.attributes.person.email")
-  = f.text_field :email, class: 'form-control'
-  = t("activerecord.attributes.person.phone")
-  = f.text_field :phone_number, class: 'form-control'
-  = t("activerecord.attributes.person.background")
-  = f.text_field :background, class: 'form-control'
-  = t("activerecord.attributes.person.roles")
-  = f.collection_check_boxes(:role_ids, Role.all, :id, :identifier) do |b|
-    = b.label { b.check_box + ' ' + t("project_roles." + b.text) }
-  %br
-  = link_to_remove_association t("remove_person"), f, class: 'remove_person fa fa-trash'
+.nested-fields.panel.panel-default.col-md-7
+  .panel-body
+    = t("activerecord.attributes.person.name")
```

Comment:
I just found out a better way to do this, that will make the label clickable:
`f.label :name`